app/assets/javascripts/admin/templates/modal/admin-install-theme.hbs

```diff
@@ -0,0 +1,122 @@
+{{#d-modal-body class='upload-selector install-theme' title="admin.customize.theme.install"}}
+  <div class="install-theme-items">
+    <div class="install-theme-item">
```

Comment:
These four `install-theme-item` parts look like a good candidate for a component